spec/topological_inventory/openshift/operations/worker_spec.rb

```diff
@@ -3,8 +3,7 @@
 RSpec.describe TopologicalInventory::Openshift::Operations::Worker do
   let(:client) { double(:client) }
 
-  describe "#run" do
-    let(:messages) { [ManageIQ::Messaging::ReceivedMessage.new(nil, "ServicePlan.order", payload, SecureRandom.uuid)] }
+  describe "#order_service" do
```

Comment:
Super-minor, but it would be helpful to say here `describe "#order_service (private)" do`.  This way it makes it easier to see why a) you're calling .send, and b) why it's just a test artifact to call thread.join

Reply:
:+1:

Reply:
Done